lib/bibliothecary/analyser.rb

```diff
@@ -160,6 +177,11 @@ def determine_kind_from_info(info)
           .fetch(:kind, nil)
       end
 
+      def determine_multiple_from_info(info)
+        first_matching_mapping_details(info)
+          .fetch(:multiple, nil)
```

Comment:
this could be "is info[:kind] a string or a list?" maybe

Reply:
probably makes it cleaner
i did it this way because we talked about being explicit in the mapping info, but this feels better to me.